docs/content/learn/tutorials/metadata/_index.md

```diff
@@ -0,0 +1,783 @@
+---
+title: Creating and Using Metadata in OSCAL
+description: A tutorial that covers the usage of the Metadata Section in OSCAL
+weight: 5
+suppresstopiclist: true
+---
+
+This tutorial covers the basics of the common Metadata Section, and provides a walk-through on creating one for a new document. Before reading this tutorial you should:
```

Comment:
I suggest adding a link to https://pages.nist.gov/OSCAL/concepts/layer/overview/, or at minimum the Metadata definition or a pointer/link to its reference. It is important for people to understand that Metadata is a global assembly found in all OSCAL moddels.

Reply:
Addressed below.